docs/source/api_reference/models.rst

```diff
@@ -465,3 +465,6 @@ MPNNModel
 
 .. autoclass:: deepchem.models.torch_models.MPNNModel
   :members:
+
+Note that this is an alternative implementation for MPNN and currently you can only import it from
```

Comment:
Can you move this line to the line above `.. autoclass:: deepchem.models.torch_models.MPNNModel`?

Reply:
Done.

Reply:
Thanks!